addons/mail/static/src/model/make_store.js

```diff
@@ -66,6 +66,17 @@ export function makeStore(env, { localRegistry } = {}) {
                             if (Model._.parentFields.has(name)) {
                                 const parentFieldName = Model._.parentFields.get(name);
                                 const parentRecordFullProxy = recordFullProxy[parentFieldName];
+                                if (!parentRecordFullProxy) {
+                                    const ParentModel =
+                                        Models[Model._.fieldsTargetModel.get(parentFieldName)];
+                                    if (
+                                        isRelation(ParentModel, name) &&
+                                        isMany(ParentModel, name)
```

Comment:
Just `isMany` is enough: `isRelation` is checking `isMany || isOne` by the way ^^